### PR TITLE
Added server port configurable functionallity

### PR DESCRIPTION
**Pull request by SietzeKoonstra**
Server:
- rmv: private readonly port;
- add: GetPort() function which gets the port from the config.
- chg: writeline now also informs about the listening port.

Configuration:
- add: writeline for new config.
- add: writer.Flush()

### CODE REVIEW

**Review by TimGels — APPROVED** [review]
lgtm

File: Server/Server.cs [hunk 1]
```diff
@@ -123,10 +131,11 @@ public void StartListening()
         {
             try
             {
+                int port = GetPort();
                 // Listen on both loopback and normal network adapters.
                 server = new TcpListener(IPAddress.Any, port);
```
Comment (JohannesKauffmann) [inline Server/Server.cs:136]: Maybe this can be one line?

**Review by JohannesKauffmann — APPROVED** [review]
Other than my comment, lgtm.

File: Server/Server.cs (hunk 1 above)
Comment (JohannesKauffmann) [reply Server/Server.cs:136]: Wait no that can't be. My bad.

**Pull request merged**